### PR TITLE
Update QUEST_JOBSTEP.tsv

### PR DESCRIPTION
**Pull request by fecono**

### CODE REVIEW

**Comment by fecono** [comment]
Reviewed and edited any mistakes I could find in section 1-420

File: QUEST_JOBSTEP.tsv [hunk 1]
```diff
@@ -393,9 +393,9 @@ QUEST_JOBSTEP_20150317_000392	If you dream to reach to a higher level as a Wugus
...
 QUEST_JOBSTEP_20150317_000394	It's hard to stray away from the charm of poison. I'm one of them. {nl}I'll let you know how appealing it is too. Let's start slowly from today on.
 QUEST_JOBSTEP_20150317_000395	It is your assignment to find the hidden monsters.{nl}The location is Purple Tree Forest.{nl}If you wish to be a Scout, you shoud be able to do this. Don't you think so?
-QUEST_JOBSTEP_20150317_000396	No one can hide perfectly from the scouts.
-QUEST_JOBSTEP_20150317_000397	You did it easier than I thought.{nl}Great. I think it'll be fun to teach you. {nl}Come to me anytime you want to learn from me.
-QUEST_JOBSTEP_20150317_000398	I think you're good enough to be accepted as my disciple but you still need to be tested. {nl}I will welcome you if you bring the rubbings of Lydis Schaffen's gravestone in the courtyard of the Goddess.
+QUEST_JOBSTEP_20150317_000396	No one can hide perfectly from the scouts. did it easier than I thought.{nl}Great. I think it'll be fun to teach you. {nl}Come to me anytime you want to learn from me.
+QUEST_JOBSTEP_20150317_000398	I think you're good enough to be accepted
```
Comment (ensata) [inline QUEST_JOBSTEP.tsv]: Line switch and a sliced sentence. Watch your fingers with the hotkeys.

File: QUEST_JOBSTEP.tsv [hunk 2]
```diff
@@ -393,9 +393,9 @@ QUEST_JOBSTEP_20150317_000392	If you dream to reach to a higher level as a Wugus
...
 QUEST_JOBSTEP_20150317_000395	It is your assignment to find the hidden monsters.{nl}The location is Purple Tree Forest.{nl}If you wish to be a Scout, you shoud be able to do this. Don't you think so?
-QUEST_JOBSTEP_20150317_000396	No one can hide perfectly from the scouts.
-QUEST_JOBSTEP_20150317_000397	You did it easier than I thought.{nl}Great. I think it'll be fun to teach you. {nl}Come to me anytime you want to learn from me.
-QUEST_JOBSTEP_20150317_000398	I think you're good enough to be accepted as my disciple but you still need to be tested. {nl}I will welcome you if you bring the rubbings of Lydis Schaffen's gravestone in the courtyard of the Goddess.
+QUEST_JOBSTEP_20150317_000396	No one can hide perfectly from the scouts. 
+QUEST_JOBSTEP_20150317_000398	You did it easier than I thought.{nl}Great. I think it'll be fun to teach you. {nl}Come to me anytime you want to learn from me.
+QUEST_JOBSTEP_20150317_000397	I think you're good enough to be accepted as my disciple but you still need to be tested. {nl}I will welcome you if you bring the rubbings of Lydis Schaffen's gravestone in the courtyard of the Goddess.
```
Comment (ensata) [inline QUEST_JOBSTEP.tsv]: The line code was switched, too. You should inspect the diff.

**Comment by imcgames** [comment]
Thank you :+1:

**Pull request merged**